Checkout/Checkout/Integration/StubProvider.swift

```diff
@@ -195,12 +195,12 @@ enum StubProvider {
   private static let tokenData = """
     {
         "version": "EC_v1",
-        "data": "cWjjsdADd0e6O2cba0cpF1458RnoR0DAxvP/lfwBPz/HfcfBQY8Wld/m+k8WjBpwn504khNVFuJ3pGdZwUHUGVv5PrujtiglkCH3B+uMDYSzalfCi6/wQf0zDmHJVoghZb0dDz4Xrh4NicTYqcmTBzPRSEcisDxxbyzH8nD2jsd/bBA2Q+jzCGhWcV/gKKjLY2XQOEc0RpHyVAKtanGSfCpCZQPQ6D/19nzvWQe9kuqfLPqmB+gBi6Z7eYMhgdLKUZpXd/m3TL3AjrOeZxvoUr++VEI+XqRqDJT5GH+cmKRlR4/ezM9y4fwdO6DJUmh8kx1iWh0CFO38KmPM+dEcL7vKS2UkIyw4FExt4LLtgGBF2P6xu7JWUyRGnLYXRfZxpd0VpgGqCqLcDRn1",
```

Comment:
Unrelated to this PR, but blocker for pipeline. Too many tasks at the moment to separate.

Reply:
Was the data expired ? What was the reason to update this.

Reply:
Yes, the data contained was of a test Apple Token, which comes with certain information, part of which is an expiry date. That makes it un-processable to our backend which then continues to fail on our "integration tests", blocking the pipelines